catalog/ui/src/app/Catalog/CatalogItemDetails.tsx

```diff
@@ -127,6 +130,11 @@ const CatalogItemDetails: React.FunctionComponent<CatalogItemDetailsProps> = ({
     }
   }
 
+  async function requestWorkshop(): Promise<void> {
```

Comment:
I think this could be a regular function (without the async and the Promise)

Reply:
Good catch, thanks!